dot-net-sdk/dto/AssignmentLogData.cs

```diff
@@ -35,4 +36,16 @@ public record AssignmentLogData
         MetaData = metaData;
         ExtraLogging = extraLoggging;
     }
+    public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
super minor but should there a blank line between this and the previous method?

Reply:
📇